parm/metplus/GridStat_ensprob_RETOP.conf

```diff
@@ -105,149 +103,130 @@ GRID_STAT_DESC = NA
+OBS_VAR4_OPTIONS = censor_thresh = lt-20.0;
+                   censor_val = -20.0;
+                   cnt_thresh = [ >15 ];
+                   cnt_logic = UNION;
+                   convert(x) = x * 3280.84 * 0.001;
 
+#
 #Process as scalars for neighborhood methods
 ## Note that the number of forecast and obs thresholds must match
 ## but won't actually be applied to NBRCNT outputs with "nbrhd.field = OBS;"
-#FCST_VAR5_NAME = RETOP_L0_ENS_FREQ_ge20
+#
 FCST_VAR5_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge20
 FCST_VAR5_LEVELS = L0
 FCST_VAR5_THRESH = ==0.1
 FCST_VAR5_OPTIONS = prob = FALSE;
-
 OBS_VAR5_NAME = EchoTop18
 OBS_VAR5_LEVELS = Z500
-OBS_VAR5_THRESH = >=20.0
-OBS_VAR5_OPTIONS = censor_thresh = lt-20.0; censor_val = -20.0; cnt_thresh = [ >15 ]; cnt_logic = UNION; convert(x) = x * 3280.84 * 0.001; nbrhd = { field = OBS; shape = SQUARE; width = [ 1 ]; vld_thresh = 1.0; }
+OBS_VAR5_THRESH = ge20
+OBS_VAR5_OPTIONS = censor_thresh = lt-20.0;
+                   censor_val = -20.0;
+                   cnt_thresh = [ >15 ];
+                   cnt_logic = UNION;
+                   convert(x) = x * 3280.84 * 0.001;
+                   nbrhd = { field = OBS; shape = SQUARE; width = [ 1 ]; vld_thresh = 1.0; }
 
-#FCST_VAR6_NAME = RETOP_L0_ENS_FREQ_ge30
 FCST_VAR6_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge30
 FCST_VAR6_LEVELS = L0
 FCST_VAR6_THRESH = ==0.1
 FCST_VAR6_OPTIONS = prob = FALSE;
-
 OBS_VAR6_NAME = EchoTop18
 OBS_VAR6_LEVELS = Z500
-OBS_VAR6_THRESH = >=30.0
-OBS_VAR6_OPTIONS = censor_thresh = lt-20.0; censor_val = -20.0; cnt_thresh = [ >15 ]; cnt_logic = UNION; convert(x) = x * 3280.84 * 0.001; nbrhd = { field = OBS; shape = SQUARE; width = [ 1 ]; vld_thresh = 1.0; }
+OBS_VAR6_THRESH = ge30
+OBS_VAR6_OPTIONS = censor_thresh = lt-20.0;
+                   censor_val = -20.0;
+                   cnt_thresh = [ >15 ];
+                   cnt_logic = UNION;
+                   convert(x) = x * 3280.84 * 0.001;
+                   nbrhd = { field = OBS; shape = SQUARE; width = [ 1 ]; vld_thresh = 1.0; }
 
-#FCST_VAR7_NAME = RETOP_L0_ENS_FREQ_ge40
 FCST_VAR7_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge40
 FCST_VAR7_LEVELS = L0
 FCST_VAR7_THRESH = ==0.1
 FCST_VAR7_OPTIONS = prob = FALSE;
-
 OBS_VAR7_NAME = EchoTop18
 OBS_VAR7_LEVELS = Z500
-OBS_VAR7_THRESH = >=40.0
-OBS_VAR7_OPTIONS = censor_thresh = lt-20.0; censor_val = -20.0; cnt_thresh = [ >15 ]; cnt_logic = UNION; convert(x) = x * 3280.84 * 0.001; nbrhd = { field = OBS; shape = SQUARE; width = [ 1 ]; vld_thresh = 1.0; }
+OBS_VAR7_THRESH = ge40
+OBS_VAR7_OPTIONS = censor_thresh = lt-20.0;
+                   censor_val = -20.0;
+                   cnt_thresh = [ >15 ];
+                   cnt_logic = UNION;
+                   convert(x) = x * 3280.84 * 0.001;
+                   nbrhd = { field = OBS; shape = SQUARE; width = [ 1 ]; vld_thresh = 1.0; }
 
-#FCST_VAR8_NAME = RETOP_L0_ENS_FREQ_ge50
 FCST_VAR8_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge50
```

Comment:
We should maybe consider GE0 verification here too...though I can't remember if this variable is above sea level or above ground level

Reply:
@mkavulich What is GE0?